newIDE/app/src/MainFrame/index.js

```diff
@@ -757,6 +758,7 @@ const MainFrame = (props: Props) => {
           fileMetadata,
           storageProvider,
           storageProviderOperations,
+          authenticatedUser,
```

Comment:
Why add authenticated user to dependencies of this function?

Reply:
In the case of being on web, adding a private asset will only be allowed if you've saved your project on the cloud.
This function will then upload all the resources to the cloud, and the authenticatedUser is required to generate pre-signed URLs with authorization.
(see CloudResourceMover, soon moved to CloudResourceFetcher)

Reply:
Oh right, my bad, I mixed lines up